ui/src/components/customModal/ResetPwdModal.tsx

```diff
@@ -108,6 +108,57 @@ export default function ResetPwdModal({
                 defaultMessage: '请输入',
               }),
             },
+            () => ({
```

Comment:
Why not use `passwordRules` here?

Reply:
There are many original rules and may not be compatible

Reply:
`passwordRules` are actually a subset of this list. Use `...passwordRules` here for reusing.

Reply:
done